device_doctor/lib/src/config.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+
+import 'device.dart';
+
+Config _config;
+Config get config => _config;
+
+/// Configures device operating system based on injected decice OS.
+///
+/// This determines the [DeviceDiscovery].
+class Config {
```

Comment:
This doesn't need to be addressed this PR, but we should consider passing a device ip instead of assuming there is only ever going to be one device attached.

Reply:
Yeah, I need to check swarming bot_config to see what we can use here, or maybe device id. Will include that once confirmed.